src/main/java/de/tum/in/www1/artemis/web/rest/StudentExamResource.java

```diff
@@ -263,7 +263,7 @@ public ResponseEntity<StudentExam> getStudentExamForConduction(@PathVariable Lon
 
         // students can not fetch the exam until 5 minutes before the exam start, we use the same constant in the client
         if (ZonedDateTime.now().plusMinutes(EXAM_START_WAIT_TIME_MINUTES).isBefore(studentExam.getExam().getStartDate())) {
-            throw new AccessForbiddenException();
+            throw new AccessForbiddenException("Students cannot download the student exams until 5 minutes before the exam start");
```

Comment:
```suggestion
            throw new AccessForbiddenException("Students cannot download the student exams until " + EXAM_START_WAIT_TIME_MINUTES + " minutes before the exam start");
```

Reply:
If we want to apply this change, we should probably also change it in the comment in line 264